src/components/menu/components/menuLinks/MenuLinks.js

```diff
@@ -0,0 +1,70 @@
+//Libraries
+import React from 'react';
+import PropTypes from 'prop-types';
+
+//Helpers
+import _map from 'lodash/map';
+import { isEmpty } from 'lodash';
+
+//constants
+import { INITIAL_STATE } from './constants/menuLinks.general';
+
+//Components
+import LinkItem from '../../../../commonComponents/linkItem';
+
+//CSS
+import './menuLinks.css';
+
+class MenuLinks extends React.Component {
+  state = INITIAL_STATE;
+
+  componentDidMount() {
+    const {
+      location: { hash },
+    } = window;
+    const { categories } = this.props;
+    const currActive = isEmpty(hash) ? categories[0].name : hash.slice(1);
+    this.setState({
+      currActive: currActive,
+    });
+  }
+
+  setCurrentActiveLink = (newRef) => {
+    const newActive = newRef.current.hash.substring(1);
+    this.setState({ currActive: newActive });
+  };
+
+  renderMenuLinkItem = (category) => {
+    const { currActive } = this.state;
+    const isEqual = category === currActive;
+    const linkRef = React.createRef();
+    const categoryHref = `#${category}`;
```

Comment:
you can create this href in link component

Reply:
If we did there then it will not be re usable